INSTALL.md

```diff
@@ -92,9 +92,10 @@ Prerequisites
     This is only needed where it exists, so a warning about the package not being
     found is not a problem.
 
-    Next you need Yarn to install the JS dependencies. There are a variety of
-    installation methods described on their website, located here:
-    https://yarnpkg.com/en/docs/install
+    Next you need Yarn to install the JS dependencies. This can be obtained via
+    corepack, which is bundled with Node.js:
+
+        sudo corepack enable
```

Comment:
"You can also specify the installation directory via `--install-directory`. If Yarn v1 is still required on the system, that's probably necessary" from the PR comments - should this be documented here then? How common is yarn v1 nowadays?

Reply:
> How common is yarn v1 nowadays?

I have no idea, but v1 is the only version available on npm or apt, so I imagine quite common still